### PR TITLE
Fixed bug with ToyModelODE when t=0 wasn't included in requested time points

### PR DESCRIPTION
**Pull request by MichaelClerx**
Fix to be merged into #1038

### CODE REVIEW

**Comment by MichaelClerx** [comment]
Hey @ben18785 ! This is a proposed fix. If you merge it the other branch should start to work again :D

File: pints/tests/test_toy_fitzhugh_nagumo_model.py [hunk 1]
```diff
@@ -65,19 +65,19 @@ def test_sensitivities(self):
...
+        self.assertAlmostEqual(sens[1, 0, 2], 5.01378, 5)
+        self.assertAlmostEqual(sens[2, 1, 1], 0.82883, 4)
 
         # Test without initial point in range
         sols, sens = model.simulateS1(parameters, [7, 12])
-        print(sens)
-        print(sens[:, 0][:, 2])
-        self.assertAlmostEqual(sens[:, 0][:, 2][0], 5.01378, 5)
-        self.assertAlmostEqual(sens[:, 1][:, 1][1], 0.82883, 4)
+        self.assertAlmostEqual(sens[0, 0, 2], 5.01378, 5)
```
Comment (ben18785) [inline pints/tests/test_toy_fitzhugh_nagumo_model.py]: Have you changed the shape of sensitivities that are returned? Or is this just aother way to refer to them? If the former, then we need to check that the notebooks (F-N, Goodwin and Lotka-Volterra) still work with the gradient-based samplers (they weren't before as the sensitivities were wrong).

Comment (MichaelClerx) [reply pints/tests/test_toy_fitzhugh_nagumo_model.py]: This is just a simpler way to say the same thing!

Old code says:
- Take a 3d array, turn it into a 2d array, by selecting only bits where 2nd indice is 2 (first brackets)
- Now make a 1d array by selecting only bits where 2nd indice (old 3d indice) is 2
- Now select a value from the resulting 1d array

It's much easier to just immediatly select the item you want :D

Comment (ben18785) [reply pints/tests/test_toy_fitzhugh_nagumo_model.py]: Cool -- yep, looks neater! Thanks -- feel free to merge!

Comment (ben18785) [reply pints/tests/test_toy_fitzhugh_nagumo_model.py]: Actually, I just did it! (I didn't think I was allowed to merge.)

**Pull request merged**